<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cesta_cli</class>
 <widget class="QWidget" name="cesta_cli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="NameClient">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF do Cliente:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_client">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="retorno_bd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nome Produto</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Qnt_item</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pesquisar.png</normaloff>pesquisar.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
